BE/src/main/java/team18/airbnb/domain/AccommodationFee.java

```diff
@@ -0,0 +1,57 @@
+package team18.airbnb.domain;
+
+import java.time.Duration;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+
+import javax.persistence.Embeddable;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+
+import lombok.Getter;
+
+@Embeddable
+@Getter
+public class AccommodationFee {
+
+    // 1박 금액
+    private int amountOfDay;
+
+    // 일 수에 따른 총 금액
+    private int totalAmountOfDay;
+
+    // 청소비
+    private int cleaningFee;
+
+    // 서비스 수수료
+    private int serviceFee;
+
+    // 숙박세와 수수료
+    private int roomCharge;
+
+    // 총 예약 확정 금액
+    private int totalAmountOfReservation;
+
+    // 주 단위 요금 할인
+    @Enumerated(EnumType.STRING)
+    private DiscountPolicy discountPolicy;
+
+    public double calculationTotalAmountOfDay(LocalDate checkinTime, LocalDate checkoutTime, Accommodation accommodation) {
```

Comment:
1. 메서드의 이름의 시작은 가급적 동사로 시작하도록 해주세요!
2. accomodationFee 안에 정의된 메서드이니 accomodation을 굳이 받아서 다시 accomodationFee를 꺼낼 필요가 없겠죠?

```suggestion
    public double calculateTotalAmount(LocalDate checkinTime, LocalDate checkoutTime) {
```

Reply:
넵 알겠습니다. 바로 변경하겠습니다.
다른 클래스에서 계산을 사용하는 로직을 그대로 복붙하여, Accommodation이 포함된거 같습니다.
중복된 로직이라 합치고 싶지만, 현재 DB 설계상 나눌수 밖에 없어 저희도 고민을 좀 더 해보도록 하겠습니다.

Reply:
db 설계를 논외하고도 Accomodation 없이 계산이 가능할것같아요 :)